starcli/__main__.py

```diff
@@ -19,12 +20,19 @@
 
 
 # could be made into config option in the future
-CACHED_RESULT_PATH = xdg_cache_home() / "starcli.json"
+CACHED_RESULT_PATH = Path(BaseDirectory.xdg_cache_home) / "starcli.json"
```

Comment:
This one is weird, but basically doing `pip3 install -r requirements_dev.txt` installs a version of `xdg` that does not export `xdg_cache_home` as is. Possibly a breaking change in some of the minor/patch version of `xdg`?  I've double-checked it with fresh `venv`.

Reply:
That's weird, I've tested with python 3.8 and 3.10, on both ubuntu and arch. Are you having this issue by installing from PyPI as well?

Reply:
Hmm, it seems like you might be using [pyxdg](https://www.freedesktop.org/wiki/Software/pyxdg/), as [xdg](https://pypi.org/project/xdg/) doesn't have `BaseDirectory`.